public/stylesheet.css

```diff
@@ -190,14 +193,20 @@ body {
   color: var(--color-text-default);
   display: flex;
   flex-direction: column;
+  line-height: 1;
```

Comment:
This rule causes the text to be a bit close together compared to the current design (comparison below). I suggest we deviate from Mayer's reset and, based on [the MDN line-height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values) ("`normal`: [...] use a default value of roughly `1.2`") and some local testing, set this to `1.4`

```suggestion
  line-height: 1.4;
```

### Example comparison

| This Pull Request | Live site | `line-height: 1.4` |
| --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/3742559/126155840-702d25e2-e317-4ffa-b403-400ef026f783.png) | ![image](https://user-images.githubusercontent.com/3742559/126155810-9959a8f0-1f5e-4e33-8612-c88c037ddcc4.png) | ![image](https://user-images.githubusercontent.com/3742559/126155964-6c4958e6-e2b1-4499-923b-390228595814.png) |

Reply:
Yes,1.4 is good, added

Reply:
The page is too long, I wonder if anybody would ever scroll down to footer, I think the content needs to be paginated.

Reply:
> Yes,1.4 is good, added

Thanks!


> The page is too long, I wonder if anybody would ever scroll down to footer, I think the content needs to be paginated.

Not sure what you mean by "too long", but pagination is something we could consider. However, that's outside the scope of this Pull Request, feel free to open an issue for it though :slightly_smiling_face: 

The main challenge with pagination will be that we don't have a back-end and that we still need to support search over all icons - not impossible just something to consider. Also, I'm personally in favor of having one long page, though improving initial load time is a concern that pagination would solve - though I'd personally prefer "full" lazy loading.